skimage/external/test_tifffile.py

```diff
@@ -17,15 +17,15 @@
 np.random.seed(0)
 
 
-@skipif(si is None)
+@pytest.mark.skipif(si is None, reason="skimage not installed")
```

Comment:
If you absolutely require this, I'll do it, but I'd rather not.
I'm tired of pytest already.